transports/msmq/distributor/configuration.md

```diff
@@ -74,4 +75,4 @@ If self-hosting the endpoint here is the code required to enlist the endpoint wi
 
 snippet: ConfiguringWorker
 
-Similar to self hosting, ensure the `app.config` of the worker contains the `MasterNodeConfig` section to point to the host name where the distributor process is running.
+Similar to self-hosting, ensure the `app.config` of the worker contains the `MasterNodeConfig` section to point to the hostname where the distributor process is running.
```

Comment:
> Similar to self hosting, ensure the `app.config` of the worker contains the `MasterNodeConfig` section

I think configuring that in app.config  is redundant?  https://docs.particular.net/transports/msmq/distributor/configuration#versions-nservicebus-6-0-and-above